modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/common/components/product-comparison/index.js

```diff
@@ -170,11 +173,19 @@ const ProductComparison = ({
 					</div>
 				</div>
 
-				<div className="quote-body">
+				<div
+					className={classNames('quote-body', {
+						selectQuoteMobile,
+					})}
+				>
 					<ListItems {...productDetails} />
 				</div>
 
-				<div className="border-0 p-0">
+				<div
+					className={classNames('border-0 p-0', {
+						selectQuoteMobile,
```

Comment:
className shouldn't be camelCase, use hyphen-case

```suggestion
 'select-quote-mobile': selectQuoteMobile
```